### PR TITLE
German translation/docs/tutorial/index

### PR DESCRIPTION
**Pull request by rogerbrinkmann**
German translation for /docs/de/tutorial/index.md

### CODE REVIEW

File: docs/de/docs/tutorial/index.md [hunk 1]
```diff
@@ -0,0 +1,80 @@
+# Tutorial - Anleitung - Einstieg
+
+Dieses Tutorial zeigt Ihnen Schritt für Schritt, wie Sie **FastAPI** mit den meisten seiner Funktionen nutzen können.
+
+Jeder Abschnitt baut schrittweise auf den vorhergehenden auf, ist aber nach einzelnen Themen gegliedert, so dass Sie direkt zu einem bestimmten Thema gehen können, um Ihre speziellen API-Anforderungen zu lösen.
```
Comment (0xflotus) [inline docs/de/docs/tutorial/index.md:5]: Jeder Abschnitt baut schrittweise auf de**m** vorhergehenden auf [...]

File: docs/de/docs/tutorial/index.md [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+
+    ```
+    pip install "uvicorn[standard]"
+    ```
+
+    Dasselbe gilt für jede der optionalen Abhängigkeiten, die Sie verwenden möchten.
+
+## Anleitung für Fortgeschrittene
+
+Es gibt auch ein **Anleitung für Fortgeschrittene**, das Sie später nach diesem **Tutorial - Anleitung** lesen können.
```
Comment (0xflotus) [inline docs/de/docs/tutorial/index.md:74]: Es gibt auch eine **Anleitung für Fortgeschrittene**, die Sie später nach diesem **Tutorial - Anleitung** lesen können.

**Review by 0xflotus — CHANGES_REQUESTED** [review]

**Review by 0xflotus — APPROVED** [review]

**Comment by 0xflotus** [comment]
But I don't even know how `mkdocs` works, so I'm not sure with the [mkdocs.yml](https://github.com/tiangolo/fastapi/pull/4517/files#diff-39af450f1cf88671e7d618429a8329259a2e3bac0de8c6fc1ab954889f41d298) changes.

**Comment by rogerbrinkmann** [comment]
> But I don't even know how `mkdocs` works, so I'm not sure with the [mkdocs.yml](https://github.com/tiangolo/fastapi/pull/4517/files#diff-39af450f1cf88671e7d618429a8329259a2e3bac0de8c6fc1ab954889f41d298) changes.

`mkdocs.yml` has to be changed for the links to work properly. Also it contains the translated menu items in the sidebar.

**Comment by 0xflotus** [comment]
> > But I don't even know how `mkdocs` works, so I'm not sure with the [mkdocs.yml](https://github.com/tiangolo/fastapi/pull/4517/files#diff-39af450f1cf88671e7d618429a8329259a2e3bac0de8c6fc1ab954889f41d298) changes.
> 
> `mkdocs.yml` has to be changed for the links to work properly. Also it contains the translated menu items in the sidebar.

But I don't even know whether it will properly work or not. 😅

**Comment by rogerbrinkmann** [comment]
> But I don't even know whether it will properly work or not. 😅

I'm pretty sure it does. Using the [live server](https://fastapi.tiangolo.com/contributing/#preview-the-result) locally, it looks good.

**Pull request closed without merging**